experiments/AMIP/modular/components/ocean/slab_seaice_init.jl

```diff
@@ -1,16 +1,16 @@
-# slab_ice
+# energy equation for sea ice
 
 # sea-ice parameters
 struct IceSlabParameters{FT <: AbstractFloat}
-    h::FT # sea ice height
+    h::FT
     ρ::FT
     c::FT
     T_base::FT
     z0m::FT
     z0b::FT
-    T_freeze::FT # temperature at freezing point [K]
-    k_ice::FT   # thermal conductivity of ice [W / m / K]
-    α::FT # albedo
+    T_freeze::FT
+    k_ice::FT
+    α::FT
```

Comment:
I think it would be helpful to have comments here explaining what each of these fields is (like you have below when you call `IceSlabParameters`). It might be nicer to have them here to refer back to instead of an instance of the struct being created, which may be moved/changed in the future.